### PR TITLE
support for newer (>1.0) version of pydicom package

### PR DESCRIPTION
**Pull request by grzanka**
which require new way of importing

### CODE REVIEW

**Comment by grzanka** [comment]
Most of the problem fixed, but one funny remain.

```
..
======================================================================
ERROR: test_create_voi_2 (__main__.TestVdx)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/test_vdx.py", line 220, in test_create_voi_2
    v = create_voi_from_cube(c, name="cube4")
  File "/home/grzanka/PycharmProjects/pytrip/pytrip/vdx.py", line 501, in create_voi_from_cube
    import matplotlib._cntr as cntr
ModuleNotFoundError: No module named 'matplotlib._cntr'
```

I've checked on my local PC that matplotlib 2.2.0rc1 released on 14.02.2017 is missing a hidded feature of contour lines extraction as described here: https://stackoverflow.com/questions/18304722/python-find-contour-lines-from-matplotlib-pyplot-contour

```
→ python3                                                                                                                            ~/PycharmProjects/pytrip
Python 3.6.4 (default, Jan  5 2018, 02:35:40) 
[GCC 7.2.1 20171224] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from matplotlib import pyplot as plt
>>> from matplotlib import _cntr as cntr
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name '_cntr'
>>> from matplotlib import __version__
>>> __version__
'2.2.0rc1'
```

**Comment by grzanka** [comment]
It looks this feature was removed ion Sep 22, 2017:  https://github.com/matplotlib/matplotlib/commit/e95572c68529c9c2af743805d7e514b944759646

and will not be available starting from matplotlib 2.2.0

File: pytrip/ctx.py [hunk 1]
```diff
@@ -118,7 +118,9 @@ def create_dicom(self):
             if ds.SOPInstanceUID.startswith('2.25.'):
```
Comment (nbassler) [inline pytrip/ctx.py]: maybe this part deserves more explanation why it is needed

Comment (grzanka) [reply pytrip/ctx.py]: I agree, but anyway DICOM related stuff needs to be refactored

**Pull request merged**